src/metabase/api/dataset.clj

```diff
@@ -123,7 +123,9 @@
   (let [query (json/parse-string query keyword)]
     (api/read-check Database (:database query))
     (as-format export-format
-      (qp/process-query-and-save-execution! (dissoc query :constraints)
+      (qp/process-query-and-save-execution! (-> query
+                                                (dissoc :constraints)
+                                                (assoc :middleware {:skip-results-metadata? true}))
```

Comment:
Seems like we want `(assoc-in [:middleware :skip-results-metadata?] true)` as there could be other keys in `:middleware` right?